domain/src/main/java/com/worldline/domain/interactor/CompletableInteractor.kt

```diff
@@ -29,12 +30,13 @@ abstract class CompletableInteractor(private val executor: Executor,
 
                 }))
 
-        return completable
+        return completableWithSchedulers
     }
 
     fun clear() {
         compositeDisposable.clear()
     }
 
-    abstract fun buildCompletable(): Completable
```

Comment:
You are not calling 'execute' method inside this class. Can you provide a sample of how to implement a concrete completable use case like 'InitAppUseCase'?